.github/workflows/a11y_tests.yml

```diff
@@ -37,9 +37,6 @@ jobs:
     env:
       PLAYWRIGHT_BASE_URL: ${{ github.event.inputs.springfield_service_hostname || 'https://dev.springfield.nonprod.webservices.mozgcp.net' }}
       CI: true
```

Comment:
FYI the runner sets [`CI=true`](https://docs.github.com/en/actions/reference/workflows-and-actions/variables#default-environment-variables) itself by default, so it's superfluous here.

I haven't removed it as it might be used in Playwright configs to make decisions, so it's a little more explicit that such value is available in `process.env.CI` conditionals for visibility.